README.md

```diff
@@ -1 +1,107 @@
-# appointment-diary-front-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+
+# Book Vehicle
+
+> The Book Vehicle is a a React, Redux application based on the [back-end API-docs](https://book-vehicle.herokuapp.com/).
+
+- In this application we're fetching the returned data from the API in the Redux store.
+
+- The user can reserve a vehicle and it'll update the state automatically, , and also the user can add a vehicle in this application.
+
+## Built With
+
+- React
+- Redux
+- React Bootstrap
+- HTML
+- CSS
+- Javascript
+- node-modules
+- lighthouse
+- hintweb
+- stylint
+- eslint
+- webpack
+- web server
+- plugins for HTML and CSS
+
+## Live Demo
+
+- [Live Demo Link]()
+
+## Link to Back-end ripo
+- https://github.com/Fatima-hub333/appointment-diary-backend
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+- Copy this link `https://github.com/Fatima-hub333/appointment-diary-front-end.git`.
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `git clone https://github.com/Fatima-hub333/appointment-diary-front-end.git`.
+- Go to the repository folder in your command prompt, run `cd space-travellers-hub`.
+- Run `npm install`
+
+### Prerequisites
+
+1. NodeJS
+2. NPM
+
+### Build
+
+Type these commands into the terminal:
+
+- npm run build
+- Open dist/index.html in your web browser.
+
+### Run the application in local environment
+
+Type this command in your terminal
+
+- npm run start
+- Your web browser will open on localhost
+
+### Run tests
+
+Type this command into the terminal:
+
+- npm run test

```

Comment:
- [ ] I couldn't find any tests on your frontend project.  Please, kindly add tests for your components and Redux.  

That way, the integrity of your app over time will be preserved 💪